docs/getting-started.md

```diff
@@ -1,5 +1,14 @@
 # Getting Started with Monokle
 
+## Prerequisites
```

Comment:
move this section to be after the initial text - i.e. before the "Running from source" section